.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.analysis.typeCheckingMode": "basic"
+}
```

Comment:
This shouldn't be committed to the repo. I suggest removing it and then adding
`.vscode` to you gitignore.

Reply:
Done